browser/src/layer/tile/CalcTileLayer.js

```diff
@@ -878,6 +879,9 @@ L.CalcTileLayer = L.CanvasTileLayer.extend({
 				}
 			}
 		}
+        else if (e.commandName === '.uno:ViewColumnRowHighlighting') {
```

Comment:
maybe we can use online-only setting for that as Gokay noticed?

example with user preferences which can be remembered in the browser: https://github.com/CollaboraOnline/online/blob/875c3b0e81f2f16d639d07051f8f6f570b38ae13/browser/src/map/handler/Map.Feedback.js#L48